drivers/FlashIAP.cpp

```diff
@@ -31,6 +33,8 @@ namespace mbed {
 
 SingletonPtr<PlatformMutex> FlashIAP::_mutex;
 
+#define FLASHIAP_BLANK 0xFF
```

Comment:
Many of the STM32L devices have an erased value of 0x00 rather than 0xFF. (the only reason I know this is because I came across it today while adding DAPLink support to the STM32L151CBU6A)  The HAL API should probably to be modified to expose this information if it is used as this level.

In this case it looks like this value may be used just as a pad value. If so you might want to rename it so it isn't confused with the erased/blank value.

Reply:
This is indeed used just once as a PAD value in the driver, so the define will be removed. 
The driver doesn't assume that blank values are 0xFF, but other code (such as NVStore) does, so will ask for such an API in HAL. If and when implemented, it should be exposed via this driver.